src/fsharp/service/ServiceParamInfoLocations.fsi

```diff
@@ -33,8 +36,13 @@ type public FSharpNoteworthyParamInfoLocations =
     member IsThereACloseParen : bool   
 
     /// Either empty or a name if an actual named parameter; f(0,a=4,?b=None) would be [|None; Some "a"; Some "b"|]
-    member NamedParamNames : string option []  
+    member NamedParamNames : string option []
+
+    /// Array of locations for each argument, and a flag if that argument is named
+    member ArgLocations: TupledArgumentLocation []
```

Comment:
We've discussed this offline - I guess I would expect this to become `CurriedArgumentLocations`.

Please also settle on either `Arg` or `Argument` within

Reply:
In this case they're not the curried argument locations, since these are found when looking at a tupled invocation like so:

```fsharp
let x = M((* here*) x, y)
```